src/components/AddPet/index.js

```diff
@@ -0,0 +1,105 @@
+import { Component } from 'pet-dex-utilities';
+
+import Sliding from '../Sliding';
+import ProgressBar from '../ProgressBar';
+
+import PetVetPage from '../../layouts/app/pages/PetVet';
+import PetWeight from '../../layouts/app/pages/PetWeight';
+import PetRegister from '../../layouts/app/pages/PetRegister';
+
+import arrowLeft from './assets/arrow-left.svg';
+
+import './index.scss';
+
+const cards = [];
```

Comment:
po, cade o cardzinho do pet? tinha um, eu não to doido, volta com ele tadinho pq se não n da nem pra testar a page no storybook 
![image](https://github.com/user-attachments/assets/ceb64312-0475-4add-9ade-bc26392f2f6c)

Reply:
é pq aí tinham q ter todos os q vao ficar oficialmente no cadastro, eu coloquei pra testar e havia esquecido de tirar